util/gh-pages/index_template.html

```diff
@@ -49,7 +49,7 @@
     <script src="theme.js"></script> {# #}
 
     <div class="container"> {# #}
-        <h1 class="page-header">Clippy Lints <span id="lint-count" class="badge"></span></h1> {# #}
+        <h1 class="page-header">Clippy Lints <span class="badge">Total number: {{ count }}</span></h1> {# #}
```

Comment:
The space between ":" and the lint count seem to not be preserved when rendering.


```suggestion
        <h1 class="page-header">Clippy Lints <span class="badge">Total number:&nbsp;{{ count }}</span></h1> {# #}
```